src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallsController.kt

```diff
@@ -48,15 +55,28 @@ class RecallsController(
   @Autowired private val userDetailsService: UserDetailsService,
   @Autowired private val recallService: RecallService,
   @Autowired private val prisonValidationService: PrisonValidationService,
-  @Autowired private val courtValidationService: CourtValidationService
+  @Autowired private val courtValidationService: CourtValidationService,
+  @Autowired private val decoder: Base64.Decoder
 ) {
 
+  @Serializable
+  data class Token(@SerialName("user_uuid") val userUuid: String)
+
   @PostMapping("/recalls")
-  fun bookRecall(@RequestBody bookRecallRequest: BookRecallRequest): ResponseEntity<RecallResponse> =
-    ResponseEntity(
-      recallRepository.save(bookRecallRequest.toRecall()).toResponse(),
+  fun bookRecall(@RequestBody bookRecallRequest: BookRecallRequest, @RequestHeader("Authorization") bearerToken: String): ResponseEntity<RecallResponse> {
+    val token = getTokenFromHeader(bearerToken)
+
+    return ResponseEntity(
+      recallRepository.save(bookRecallRequest.toRecall(token.userUuid)).toResponse(),
       HttpStatus.CREATED
     )
+  }
+
+  private fun getTokenFromHeader(bearerToken: String): Token {
```

Comment:
"Obvs" move this to a helper/the bean/don't copy to each controller - as a future refactoring.

Reply:
Also, I guess the "real" test of this is the E2Es.

Reply:
Yep,  have done so in the next commit :)